src/api/v1beta1/properties.go

```diff
@@ -400,3 +400,11 @@ func getRawImageTag(imageURI string) string {
 	splitURI := strings.Split(imageURI, ":")
 	return splitURI[len(splitURI)-1]
 }
+
+func (dk *DynaKube) MaxFailedCsiMountAttempts() int {
+	if dk.FeatureMaxFailedCsiMountAttempts() != nil {
+		return *dk.FeatureMaxFailedCsiMountAttempts()
+	}
+
+	return -1
```

Comment:
wouldn't it be cleaner if we returned the default value here? (and not handled it in the `metadata.NewDynakube()`)

Reply:
done